platform/golden/scripts/serial.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
This is one approach. May I suggest we use ansible a a provisioner? There are many advantages. It does a really great job at this. I'll submit a few PRs to show what I have been thinking.

Cheers,
-steve

Reply:
Yes, I'll take that approach going forward. I even have a nice example of enabling 'local' in image ansible, and cleaning it up as a part of the build process (e.g. install ansible on the packer built image, rather than running from a local ansible via ssh as one would normally do)

Reply:
SGTM!